sections/methodology/main.tex

```diff
@@ -1,7 +1,203 @@
+\begin{align}
+    & \alpha L_A(p_A;T_A=1;T_B=1) + (1 - \alpha) B_A(p_A;T_A=1;T_B=1) = 
+    \nonumber \\
+    & \alpha L_B(p_B;T_A=1;T_B=1) + (1 - \alpha) B_B(p_B;T_A=1;T_B=1) 
+    \label{eq:obj_distributor_1_1}
+\end{align}
+
+The values of \(p_A\) and \(p_B\) that satisfy equation 
+(\ref{eq:obj_distributor_1_1}) can be found by using Brent's bisection 
+algorithm~\cite{brent1973algorithms}.
+The calculated value of \(p_A\) corresponds to the entry on the first row and 
+first column of the routing matrix:
+
+\begin{equation}\label{eq:routing_matrix_1_1}
+    R = 
+    \begin{pmatrix}
+        p_{1,1}^A & - & - \\
+        - & - & - \\
+        - & - & - \\
+        - & - & - \\
+    \end{pmatrix}
+\end{equation}
+
+In fact all remaining values of the routing matrix can be calculated by
+solving \(N \times M\) similar equations using Brent's bisection algorithm.
+Therefore, having calculated the entire routing matrix the payoff matrices can
+be calculated.
+Consider once again the case of \(T_A=1, T_B=1\). 
+
+
+\begin{align}
+    U_{1, 1}^A = 1 -\left( 
+        \hat{P} - P(W_A(T_A=1, T_B=1) < t) 
+    \right)^2 \nonumber
+    \\
+    U_{1, 1}^B = 1 -\left( 
+        \hat{P} - P(W_B(T_A=1, T_B=1) < t) 
+    \right)^2 \label{eq:payoff_entry_1_1}
+\end{align}
+
+These are the utilities of players \(A\) and \(B\) when both players choose a
+strategy of \(T = 1\).
+Here \(R\) is the predefined waiting time target to be met by a percentage of 
+individuals \( \hat{P} \) and \(P(W_i < t)\) is properly defined in section
+\ref{sec:proportion_within_target}.
+
+\begin{equation}\label{eq:payoff_matrices_1_1}
+    A = 
+    \begin{pmatrix}
+        U_{1, 1}^A & - & - \\
+        - & - & - \\
+        - & - & - \\
+        - & - & - \\
+    \end{pmatrix}, \quad
+    B = 
+    \begin{pmatrix}
+        U_{1, 1}^B & - & - \\
+        - & - & - \\
+        - & - & - \\
+        - & - & - \\
+    \end{pmatrix}
+\end{equation}
+
+Similar to the routing matrix, the above procedure can be repeated for all 
+possible values of \(T_A\) and \(T_B\) to populate all entries of the payoff 
+matrices. 
+
+\subsection{Nash Equilibrium}\label{sec:methodology_nash_equilibrium}
+
+Having calculated the payoff matrices \(A\) and \(B\), several algorithms can 
+be used to help us find the Nash equilibrium of the game. 
+The Nash equilibrium is the point of the game were both players have no 
+incentive to deviate from their played strategies \cite{kreps1989nash}.
+In other words their chosen strategies are a best response to each other.
+
+The Lemke-Howson algorithm is used to get all Nash equilibria of the game 
+\cite{LemkeHowson}.
+Lemke-Howson uses best response polytopes to get one of the Nash equilibrium of
+the game. 
+By iterating over all possible dropped labels of the best response polytope all
+Nash equilibria can be acquired.
+All game theoretic calculations were done in Python using the Nashpy library 
```

Comment:
This is not actually true @11michalis11, it is possible for the Lemke Howson algorithm to not be able to reach some Nash equilibria (for example in the battle of the sexes).

```suggestion
Computation of Nash equilibria can be done relatively efficiently using the Lemke Howson algorithm
\cite{LemkeHowson}.
Lemke-Howson uses best response polytopes to get one of the Nash equilibrium of
the game. 
Other algorithms exists that will compute all Nash equilibria but 
for large games the computational complexity becomes problematic.
All game theoretic calculations were done in Python using the Nashpy library 
```